content/10.zk-stack/20.running/55.ownership-model.md

```diff
@@ -0,0 +1,117 @@
+| **L1/L2 DA Validators** | Ensure that pubdata is committed properly | - | L1: optionally shared, L2: no | [Default L1](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l1-contracts/contracts/state-transition/data-availability)<br/>[3rd Party L1](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/da-contracts/contracts/da-layers/)<br/>[L2](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l2-contracts/contracts/data-availability) |
+
+## Roles overview
+
+| Name | Purpose                                                                                                                                                                                                                 | Owned by | Shared | Source code                                                                                                                                             |
+| --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- | --- |---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **DecentralizedGovernance** (set of contracts) | Owns ProtocolUpgradeHandler, controls all ecosystem contracts, accepts/vetoes proposals.                                                                                                                                | - | Yes | [Repository](https://github.com/zksync-association/zk-governance/tree/96ade35e25e66d44af8736172c63aac3b5386150)                                         |
+| **ProtocolUpgradeHandler** | Ownership of CTM, Bridgehub, L1 and L2 bridges, AssetRouters, NTVs                                                                                                                                                      | DecentralizedGovernance | Yes | [Source](https://github.com/zksync-association/zk-governance/blob/96ade35e25e66d44af8736172c63aac3b5386150/l1-contracts/src/ProtocolUpgradeHandler.sol) |
+| **ML ChainAdmin** | - Revert batches within CTM<br/>- Set validator to any chain (will be removed in gateway upgrade)<br/>- Create new chain in Bridgehub<br/>- Add a base token in Bridgehub<br/>- Allow bridging to some L2SharedBridge   | ML Multisig | Yes | [Source](https://github.com/matter-labs/era-contracts/tree/0fcc77c4f49d10efaaba4808b2c9b29a6fbfb232/l1-contracts/contracts/governance/ChainAdmin.sol)   |
```

Comment:
Maybe better to rename it to `EcosystemAdmin`? yes, it does collide with the ML chain admin and you may say that ATM these are owned by the same entity, but these roles are not the same